FlipperZero/FlipperZero.xcodeproj/project.pbxproj

```diff
@@ -649,6 +655,7 @@
 					"@executable_path/../Frameworks",
 					"@loader_path/Frameworks",
 				);
+				MACOSX_DEPLOYMENT_TARGET = 11.0;
```

Comment:
This will be fixed in #28 with updating of the value on the project level, please remove explicit version set on the target level
The same on lines 685, 914 and 938